### PR TITLE
docs(bot): update reusing-existing bot help

### PR DESCRIPTION
**Pull request by YitongFeng-git**

### CODE REVIEW

File: docs/fx-core/bot-help.md [hunk 1]
```diff
@@ -28,44 +28,29 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
-3. Paste the target message endpoint in text box `Messaging endpoint`.
-![image](../images/fx-core/bot/update-message-endpoint-azure.png)
-
-After updating message endpoint is done, continue to deploy and try the remote experience.
+## How to reuse existing AAD in Toolkit v2?
+
+Suppose an existing AAD has been registered, you can reuse this AAD by configuring Toolkit project settings.
+
+### Register an AAD on Azure Portal
+1. Manually create a new App registrations on [Azure Portal](https://ms.portal.azure.com/), note down the **appid**
```
Comment (IvanJobs) [inline docs/fx-core/bot-help.md:36]: Use single form - registration.

File: docs/fx-core/bot-help.md [hunk 2]
```diff
@@ -28,44 +28,29 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
+2. Create a secret for the AAD created in step1, note down the **secret**
```
Comment (IvanJobs) [inline docs/fx-core/bot-help.md:37]: add space to step1? - step 1

File: docs/fx-core/bot-help.md [hunk 3]
```diff
@@ -28,44 +28,29 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
+![image](https://user-images.githubusercontent.com/72963627/143382469-cfcb68df-0d58-478e-9184-e4279580f987.png)
```
Comment (IvanJobs) [inline docs/fx-core/bot-help.md:38]: I'm curious about the location of the image - is it stable?

**Review by eriolchan — DISMISSED** [review]

File: docs/fx-core/bot-help.md [hunk 4]
```diff
@@ -28,44 +28,30 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
+
+Suppose an existing AAD has been registered, you can reuse this AAD by configuring Toolkit project settings.
+
+### Register an AAD on Azure Portal
+1. Manually create a new App Registration on [Azure Portal](https://ms.portal.azure.com/), note down the **appid**
+2. Create a secret for the AAD created in step 1, note down the **secret**
+
+![image](../images/fx-core/bot/AAD-secret.png)
+
+3. Fill in the appid and secret in `${ProjectFolder}/.fx/config.${env}.json`, add the `bot` section, and the settings will look like:
```
Comment (IvanJobs) [inline docs/fx-core/bot-help.md:41]: under .fx/configs/?

File: docs/fx-core/bot-help.md [hunk 5]
```diff
@@ -28,44 +28,30 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
-3. Paste the target message endpoint in text box `Messaging endpoint`.
-![image](../images/fx-core/bot/update-message-endpoint-azure.png)
-
-After updating message endpoint is done, continue to deploy and try the remote experience.
+## How to reuse existing AAD in Toolkit v2?
+
+Suppose an existing AAD has been registered, you can reuse this AAD by configuring Toolkit project settings.
+
+### Register an AAD on Azure Portal
+1. Manually create a new App Registration on [Azure Portal](https://ms.portal.azure.com/), note down the **appid**
```
Comment (IvanJobs) [inline docs/fx-core/bot-help.md:36]: Application ID? based on the label of Azure Portal.

File: docs/fx-core/bot-help.md [hunk 6]
```diff
@@ -28,44 +28,30 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
+2. Create a secret for the AAD created in step 1, note down the **secret**
```
Comment (IvanJobs) [inline docs/fx-core/bot-help.md:37]: Client secret? based on the label on Azure Portal?

**Review by IvanJobs — APPROVED** [review]

**Review by eriolchan — APPROVED** [review]

**Pull request merged**